Packs/CarbonBlackEnterpriseEDR/TestPlaybooks/Carbon_Black_Enterprise_EDR_Test.yml

```diff
@@ -0,0 +1,863 @@
+id: Carbon Black Enterprise EDR Test
+version: -1
+fromversion: 5.0.0
+contentitemfields:
+  propagationLabels:
+  - all
+name: Carbon Black Enterprise EDR Test
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: "313"
+    type: start
+    task:
+      id: "313"
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quiet: false
+  "1":
+    id: "1"
+    taskid: "314"
+    type: regular
+    task:
+      id: "314"
+      version: -1
+      name: Delete Context
+      description: Delete field from context
+      scriptName: DeleteContext
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      all:
+        simple: "yes"
+      index: {}
+      key: {}
+      keysToKeep: {}
+      subplaybook: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quiet: false
+  "2":
+    id: "2"
+    taskid: "327"
+    type: regular
+    task:
+      id: "327"
+      version: -1
+      name: List Watchlists
+      description: Retrieve all watchlists.
+      script: Carbon Black Enterprise EDR|||cb-eedr-watchlist-list
+      type: regular
+      iscommand: true
+      brand: Carbon Black Enterprise EDR
+    nexttasks:
+      '#none#':
+      - "18"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quiet: false
+  "3":
+    id: "3"
+    taskid: "328"
+    type: regular
+    task:
+      id: "328"
+      version: -1
+      name: Create new wachlist
+      description: Create a new report or classifier watchlist.
+      script: '|||cb-eedr-watchlist-create'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      alerts_enabled:
+        simple: "false"
+      classifier_key: {}
+      classifier_value: {}
+      description:
+        simple: This is a test
+      report_ids:
+        simple: A59huyinQSmAr8t1a2hpg
```

Comment:
it is better not to use hard coded values in the test, as it might fail if this report disappears somehow
can we run the watchlist commands in the test after the report creation and take that id?

Reply:
yes, changed that